### PR TITLE
Fix links in Javadoc

### PR DESCRIPTION
**Pull request by XenoAmess**
as title

### CODE REVIEW

File: spring-context/src/main/java/org/springframework/context/support/FileSystemXmlApplicationContext.java [hunk 1]
```diff
@@ -151,7 +151,6 @@ public FileSystemXmlApplicationContext(
 	 * This is consistent with the semantics in a Servlet container.
 	 * @param path the path to the resource
 	 * @return the Resource handle
-	 * @see org.springframework.web.context.support.XmlWebApplicationContext#getResourceByPath
```
Comment (sbrannen) [inline spring-context/src/main/java/org/springframework/context/support/FileSystemXmlApplicationContext.java:154]: Please revert this change.

File: spring-core/src/main/java/org/springframework/core/io/DefaultResourceLoader.java [hunk 2]
```diff
@@ -179,7 +179,6 @@ else if (location.startsWith(CLASSPATH_URL_PREFIX)) {
 	 * @return the corresponding Resource handle
 	 * @see ClassPathResource
 	 * @see org.springframework.context.support.FileSystemXmlApplicationContext#getResourceByPath
-	 * @see org.springframework.web.context.support.XmlWebApplicationContext#getResourceByPath
```
Comment (sbrannen) [inline spring-core/src/main/java/org/springframework/core/io/DefaultResourceLoader.java:182]: Please revert this change.

File: spring-r2dbc/src/main/java/org/springframework/r2dbc/connection/TransactionAwareConnectionFactoryProxy.java [hunk 3]
```diff
@@ -58,7 +58,7 @@
  * {@link ConnectionFactory}, avoiding the need to define such a proxy in the first place.
  *
  * <p><b>NOTE:</b> This {@link ConnectionFactory} proxy needs to return wrapped
- * {@link Connection}s (which implement the {@link ConnectionProxy} interface) in order
+ * {@link Connection}s (which implement the {@link org.springframework.jdbc.datasource.ConnectionProxy} interface) in order
```
Comment (sbrannen) [inline spring-r2dbc/src/main/java/org/springframework/r2dbc/connection/TransactionAwareConnectionFactoryProxy.java:61]: Please revert this change.

This is obsolete documentation which needs to be reworked separately.

File: spring-r2dbc/src/main/java/org/springframework/r2dbc/connection/TransactionAwareConnectionFactoryProxy.java [hunk 4]
```diff
@@ -86,12 +86,12 @@ public TransactionAwareConnectionFactoryProxy(ConnectionFactory targetConnection
 	/**
 	 * Delegates to {@link ConnectionFactoryUtils} for automatically participating in Spring-managed transactions.
 	 * <p>
-	 * The returned {@link ConnectionFactory} handle implements the {@link ConnectionProxy} interface, allowing to
+	 * The returned {@link ConnectionFactory} handle implements the {@link org.springframework.jdbc.datasource.ConnectionProxy} interface, allowing to
```
Comment (sbrannen) [inline spring-r2dbc/src/main/java/org/springframework/r2dbc/connection/TransactionAwareConnectionFactoryProxy.java:89]: Please revert this change.

This is obsolete documentation which needs to be reworked separately.

File: spring-core/src/main/java/org/springframework/core/env/ReadOnlySystemAttributesMap.java [hunk 5]
```diff
@@ -27,7 +27,7 @@
  * Read-only {@code Map<String, String>} implementation that is backed by system
  * properties or environment variables.
  *
- * <p>Used by {@link AbstractApplicationContext} when a {@link SecurityManager} prohibits
+ * <p>Used by {@link org.springframework.core.env.AbstractEnvironment} when a {@link SecurityManager} prohibits
```
Comment (sbrannen) [inline spring-core/src/main/java/org/springframework/core/env/ReadOnlySystemAttributesMap.java:30]:
```suggestion
 * <p>Used by {@link AbstractEnvironment} when a {@link SecurityManager} prohibits
```

File: spring-r2dbc/src/main/java/org/springframework/r2dbc/connection/TransactionAwareConnectionFactoryProxy.java [hunk 6]
```diff
@@ -86,12 +86,12 @@ public TransactionAwareConnectionFactoryProxy(ConnectionFactory targetConnection
...
 	 * retrieve the underlying target {@link Connection}.
 	 *
 	 * @return a transactional {@link Connection} if any, a new one else.
 	 * @see ConnectionFactoryUtils#doGetConnection
-	 * @see ConnectionProxy#getTargetConnection
+	 * @see org.springframework.jdbc.datasource.ConnectionProxy#getTargetConnection
```
Comment (sbrannen) [inline spring-r2dbc/src/main/java/org/springframework/r2dbc/connection/TransactionAwareConnectionFactoryProxy.java:94]: Please revert this change.

This is obsolete documentation which needs to be reworked separately.

**Review by sbrannen — CHANGES_REQUESTED** [review]
Thanks for the PR.

I've requested a few minor changes.

**Comment by XenoAmess** [comment]
@sbrannen done.

**Review by sbrannen — APPROVED** [review]

**Comment by sbrannen** [comment]
This has been merged into `master`.

Thanks

**Pull request merged**